packages/db-objects-file/src/lib/objects/objectsInMemFileDB.js

```diff
@@ -38,17 +38,21 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
                 this.log.silly(`${this.namespace} objects change: ${id} ${JSON.stringify(this.change)}`);
             };
         }
+
+        this.META_ID = '**META**';
```

Comment:
I would more do that as a normal "file" constant rather then a class property because only used here